### DISCUSSION TITLE
i cant select a fabric object when its color is transparent with out use perPixelTargetFind , i cant use perPixelTargetFind in my project because of the so many polygon objects aligned closely so i cant use that , is there any other way to select the transparent object ?

### DISCUSSION THREAD

**Discussion by SachuG3** [fabricjs/fabric.js#9874] [General]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

5.3.0

### Node Version (if applicable)

None

### Link To Reproduction

set a polygon or a object full color as transparent , i cant select the object event it has selectable true

### Steps To Reproduce

1. set a polygon or a object full color as transparent , i cant select the object event it has selectable true
2. 
3.

### Expected Behavior

set a polygon or a object full color as transparent , i cant select the object event it has selectable true

### Actual Behavior

set a polygon or a object full color as transparent , i cant select the object event it has selectable true

**Comment by asturur** [comment]
Is unclear why you can't select an object and why you want perPixelTargetFind off for transparent polygons.
Please if you want someone to engage in a conversation about your issue and help make the problem clear at first glance
